packages/js/explat/src/index.ts

```diff
@@ -20,11 +20,14 @@ declare global {
 			isEnabled: boolean;
 			enable?: ( cb: () => void ) => void;
 		};
+		_wca: {
+			push?: ( cb: () => void ) => void;
+		};
 	}
 }
 
 export const initializeExPlat = (): void => {
-	if ( window.wcTracks?.isEnabled ) {
+	if ( window.wcTracks?.isEnabled || window?._wca?.push !== undefined ) {
```

Comment:
This is used in a lot of places, should it justify a helper method? Also would be more readable than negatives and or's or and's 🤔

Reply:
Sorry, Nadir, my suggestion broke tests on `assignment-test.js` that now need updating. After this tweak, I'm ready to approve it 👍🏼